lib/models/ObjectMD.js

```diff
@@ -106,10 +106,12 @@ class ObjectMD {
             },
             'key': '',
             'location': null,
-            'isNull': '',
-            'nullVersionId': '',
-            'isDeleteMarker': '',
-            'versionId': undefined, // If no versionId, it should be undefined
+            // versionId, isNull, nullVersionId and isDeleteMarker
+            // should be undefined when not set explicitly
+            'isNull': undefined,
```

Comment:
Seems 'isNull' and 'isDeleteMarker' are booleans and could be set to false by default

Reply:
I wanted to stay consistent with what cloudserver writes when it puts a new object, which has no such attributes in the metadata.